packages/wpcom.js/src/lib/site.post.js

```diff
@@ -206,40 +201,6 @@ class SitePost {
 		const comment = new Comment( null, this._id, this._sid, this.wpcom );
 		return comment.replies( query, fn );
 	}
-
```

Comment:
It may be worth doing a broader audit for unused ones. `.related()` seems to be unused from my quick search, and I bet there are more.

Reply:
Removing also `.related` because it can be classified as a "social/engagement" feature, which this PR is focused on.